src/Tizen.WebView/Tizen.WebView/WebView.cs

```diff
@@ -328,6 +328,102 @@ public void SetFocus(bool focused)
             Interop.ChromiumEwk.ewk_view_focus_set(_realHandle, focused);
         }
 
+        /// <summary>
+        /// Get coordinates of content.
+        /// </summary>
+        /// <param name="width">The width of the coordinate.</param>
+        /// <param name="height">The height of the coordinate.</param>
+        /// <returns>'true' on success, otherwise 'false'.</returns>
+        /// <since_tizen> 6 </since_tizen>
+        public bool GetContentsSize (ref int width, ref int height)
+        {
+            return Interop.ChromiumEwk.ewk_view_contents_size_get(_realHandle, ref width, ref height);
+        }
+
+        /// <summary>
+        /// Get cookies for URL.
+        /// </summary>
+        /// <param name="url">The URL.</param>
+        /// <returns>'cookie string' on success, otherwise 'null'.</returns>
+        /// <since_tizen> 6 </since_tizen>
```

Comment:
This API is internal.
Please append EditorBrowsableState.Never.

Reply:
Done